src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy

```diff
@@ -80,7 +80,11 @@ class ApplyPatchesTask extends PatchTask {
         git.fetch('origin') >> null
         git.checkout('-B', 'master', 'origin/upstream') >> null
         git.reset('--hard') >> out
-
+        
+        if (patchDir.exists()) {
+            patchDir.mkdirs();
```

Comment:
Create the patchDir if it already exists...? Check if the condition is really necessary, will mkdirs() fail if the dir already exists?